examples/stm32/f0/stm32f0-discovery/button/button.c

```diff
@@ -62,7 +62,7 @@ int main(void)
 		gpio_toggle(GPIOC, GPIO8);
 
 		/* Upon button press, blink more slowly. */
-		if (gpio_get(GPIOA, GPIO0)) {
+		if ((gpio_get(GPIOA, GPIO0) & GPIO0) == 1) {
```

Comment:
No, this is worse, and likely to lead to errors if you try and change to say, GPIO5